README.md

```diff
@@ -186,6 +186,66 @@ Update [values.yaml](https://github.com/openclarity/apiclarity/blob/master/chart
 
 4. Open APIClarity UI in the browser: <http://localhost:8080/>
 
+## Enabling External Trace Sources Support
+Enabling external trace sources support, APIClarity can receive the trace sources from the entitites that are external to the K8s cluster. External trace sources such as Gateways, Load balancers, etc. can communicate with APIClarity to report APIs and send the traces. 
+
+The following section describes how to deploy APIClarity with the support for external trace sources
+1)	Add Helm Repo
```

Comment:
I think that numbered lists are done with `1.`, `2.` ...

Reply:
Thank you - changed the numbering format